src/components/SearchBox/SearchBox.jsx

```diff
@@ -40,6 +40,19 @@ class SearchBox extends React.Component {
     return null;
   }
 
+  /**
+   * setResearchCatalogUrl(searchString)
+   * Returns the final URL for the Research Catalog search.
+   */
+   setResearchCatalogUrl(searchString) {
+    const catalogUrl = '//www.nypl.org/research/research-catalog/search?q=';
```

Comment:
It looks like a lot of the links in the header are relative, and therefore break in localhost but the catalog search links go to `nypl.org`.  Is there something different between the catalog links and, eg, the `/research` link that requires this to happen, or is it just an ease-of-dev thing that we're not gonna be too concerned over as we're hoping to stop using this branch?

Reply:
I hoping we can make everything absolute so all links just go to the production instance for "minor" locations like other nypl.org pages. But yes, it is an ease-of-dev thing.